backend/src/login/authenticate.py

```diff
@@ -6,6 +6,8 @@
 from src.login.user.role import Role
 from src.secret import CloudSecretManager
 
+logger = logging.getLogger(__name__)
```

Comment:
This is indeed needed?  We can't keep doing `logging.warning` or do `logging.info`?

Reply:
Good catch! I'll get it fixed. It should just be `logging.info` and not use a logger.

Reply:
Fixed!